src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -433,6 +435,33 @@ public RemoteOutputsStrategyConverter() {
               + " discard the remotely cached values if they don't match the expected value.")
   public boolean remoteVerifyDownloads;
 
+  @Option(
+      name = "remote_keepalive_time",
+      defaultValue = "null",
+      documentationCategory = OptionDocumentationCategory.REMOTE,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      help =
+          "Sets the time without read activity before sending a keepalive ping. Clients must "
+              + "receive permission from the service owner before enabling this option. Keepalives "
+              + "can increase the load on services and are commonly \"invisible\" making it hard "
+              + "to notice when they are causing excessive load. Clients are strongly encouraged "
+              + "to use only as small of a value as necessary.")
```

Comment:
I think this last sentence is confusing. It would be clearer to say that "larger values for this option generate less traffic, so only use a smaller value if necessary". More explicit guidance would be even better, e.g., "don't set this to less than one minute".

Reply:
Yeah, that sounds a lot better. I took these descriptions from the documentation of the gRPC Java library and already redacted them slightly. This seems like a good improvement.